src/main/java/uk/gov/hmcts/reform/sscs/service/NotificationService.java

```diff
@@ -103,12 +103,15 @@ private boolean functionalTest(SscsCaseData newSscsCaseData) {
         return YesNo.isYes(newSscsCaseData.getFunctionalTest());
     }
 
-    private void sendSecondNotificationForLongLetters(NotificationWrapper notificationWrapper) {
+    private void sendSecondNotification(NotificationWrapper notificationWrapper) {
         if (notificationWrapper.getNotificationType().equals(ISSUE_FINAL_DECISION_WELSH)) {
             // Gov Notify has a limit of 10 pages, so for long notifications (especially Welsh) we need to split the sending into 2 parts
             notificationWrapper.getSscsCaseDataWrapper().setNotificationEventType(ISSUE_FINAL_DECISION);
             notificationWrapper.setSwitchLanguageType(true);
             sendNotificationPerSubscription(notificationWrapper);
+        } else if (notificationWrapper.getNotificationType().equals(DWP_UPLOAD_RESPONSE_NOTIFICATION)) {
+            notificationWrapper.getSscsCaseDataWrapper().setNotificationEventType(UPDATE_OTHER_PARTY_DATA);
```

Comment:
What happens here for cases that don't have other parties on them? i.e. all SSCS1 cases? Is there not a way to limit this logic so it doesn't trigger unless at least it has an other party on the case? Or do you think it's ok?

Reply:
In this scenario, there will be no any subscription to send UPDATE_OTHER_PARTY_DATA notification, so it will be called but not send any notification